src/services/LookUpService.js

```diff
@@ -5,10 +5,7 @@ import LoggingService from './LoggingService';
 const REGISTRY_URL = `${process.env.REGISTRY_URL}/lookup`;
 const logger = LoggingService.getLogger('LookUpService');
 
-const lookUpPublicKey = async (type) => {
-  const request = JSON.stringify({
-    type,
-  });
+const lookUpPublicKey = async (request) => {
```

Comment:
type was better, what does request mean. is it query by type, subscriberId, ukId

Reply:
The variable name has been changed for more readability .